types/prosemirror-commands/index.d.ts

```diff
@@ -222,17 +235,17 @@ export function chainCommands<S extends Schema = any>(
  * * **Mod-Delete** to `deleteSelection`, `joinForward`, `selectNodeForward`
  * * **Mod-a** to `selectAll`
  */
-export let pcBaseKeymap: { [key: string]: any };
+export let pcBaseKeymap: Keymap;
```

Comment:
I want to point out that this is a breaking change. After reading the README and some discussions in issues, it seems to me that it's recommended for versions of type definitions to

* always track the library version
* follow semvar

which together imply that breaking changes to the type definitions are only possible when the the corresponding library publishes a new major version. I don't agree with this policy because it means that one must willingly tolerate imprecise or plain wrong type definitions for an indefinite time (e.g. there has been no major release of `prosemirror-commands` since 1.0.0 was released in October 2017). Therefore I'm ignoring this policy and approve this breaking change.